lib/jstp.js

```diff
@@ -5,26 +5,28 @@ impress.jstp = jstp;
 
 // Create JSTP Server
 //   config <Object> Impress server worker configuration
-jstp.createServer = config => {
-  const transportModule = config.transport === 'tcp' ? 'net' : config.transport;
-  const transport = api.jstp[transportModule];
-  if (!transport) return null;
+jstp.createServer = serverConfig => {
+  const { name, transport } = serverConfig;
```

Comment:
The issue is here, `serverConfig` doesn't have the `name` field now:
```js
{ protocol: 'jstp',
  transport: 'tcp',
  address: '127.0.0.1',
  ports: [ 250 ],
  slowTime: 1000,
  bundle: false,
  nagle: false,
  timeout: 30000,
  keepAlive: 5000,
  heartbeatInterval: 0,
  key: undefined,
  cert: undefined,
  applications: undefined,
  port: 250 }
```

Reply:
@belochub , thanks, it really helped